test/conftest.py

```diff
@@ -25,6 +28,15 @@ def skipif_pre_ansible28(is_pre_ansible28):
         pytest.skip("Valid only on Ansible 2.8+")
 
 
-@pytest.fixture(scope='session')
-def test_data_dir():
-    return os.path.join(os.path.dirname(__file__), 'data')
+@pytest.fixture
+def test_data_dir(tmp_path):
+    source = Path(__file__).parent / 'fixtures' / 'data'
+    dest = tmp_path / 'data'
+    shutil.copytree(source, dest)
```

Comment:
clever.

This is function scope, and copies the entire data directory? As long as the user clears old artifacts before running, that's 14 dirs / 14 files. All of these tend to be very small in the scheme of things, so this feels like the right kind of balance to me. :+1: